src/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace GGPHP\Product\Http\Controllers;
+
+use Webkul\Product\Http\Requests\ProductForm;
+use Webkul\Product\Http\Controllers\ProductController as WebkulProductController;
+
+class ProductController extends WebkulProductController
+{
+    public function update(ProductForm $request, $id)
+    {
+        $data = request()->all();
+
+        // Set value channel is default
+        if (core()->getConfigData('catalog.products.general.channel-default')) {
+            array_merge($data, ['channel' => 'default']);
+        }
+
+        $multiselectAttributeCodes = array();
+
+        $productAttributes = $this->productRepository->findOrFail($id);
+
+        foreach ($productAttributes->attribute_family->attribute_groups as $attributeGroup) {
+            $customAttributes = $productAttributes->getEditableAttributes($attributeGroup);
+
+            if (count($customAttributes)) {
```

Comment:
Chỗ này check count() đâu cần thiết đâu tuấn

Reply:
dòng code ni là của bagisto đó a, e copy lại rồi xử lý merge giá trị channel thôi
e nghĩ chỗ này cần count() để $customAttributes có it nhất 1 phần tử thì mới thực thi code trong if